src/bundler/bundle_v2.zig

```diff
@@ -13866,10 +14410,64 @@ pub const LinkerContext = struct {
         shifts: []sourcemap.SourceMapShifts,
     };
 
+    fn mangleLocalCss(c: *LinkerContext) void {
+        if (c.has_any_css_locals.load(.monotonic) == 0) return;
+
+        const all_css_asts: []?*bun.css.BundlerStyleSheet = c.graph.ast.items(.css);
+        const all_symbols: []Symbol.List = c.graph.ast.items(.symbols);
+        const all_sources: []Logger.Source = c.parse_graph.input_files.items(.source);
+
+        // Collect all local css names
+        var sfb = std.heap.stackFallback(512, c.allocator);
+        const allocator = sfb.get();
+        var local_css_names = std.AutoArrayHashMap(bun.bundle_v2.Ref, void).init(allocator);
```

Comment:
Use std.AutoHashMap instead of ArrayHashMap, there may be many many local CSS names in one file.